tutorials/pavics/finch-intro.md

```diff
@@ -0,0 +1,6 @@
+# Finch Examples
+
```

Comment:
As suggested by Misha, possibly add a section that suggests related tutorials.

For example:  "Those who viewed these tutorials also viewed these other tutorials."